lua/docgen/renderer.lua

```diff
@@ -308,4 +309,45 @@ m.render = function(input, prefix, width)
   return table.concat(output, '\n')
 end
 
+--- renderi is a paragraph only rendering and is used for prefix indentation,
+--- beginning second line.
+---
+--- Used for parameters and field description
+m.renderi = function(input, prefix, width)
```

Comment:
Don't love the name.

Reply:
I am so bad with names :rofl: Now the name is longer than java function names :laughing: